src/host/tracing.cpp

```diff
@@ -20,6 +20,7 @@ enum TraceKeywords
     Input = 0x200,
     API = 0x400,
     UIA = 0x800,
+    CookedRead = 0x1000,
     All = 0xFFF

```

Comment:
Update all to either 0x1FFF or just 0xFFFF for future proof.

Reply:
Whoops. yes

Reply:
I think 1fff better aligns with the comment in til.h